.travis.yml

```diff
@@ -16,11 +16,6 @@ deploy:
   on:
     branch: master
 
-addons:
-  apt:
-    packages:
-      - oracle-java9-installer
-
```

Comment:
Seems like this should have been done with #119 ;-) Please move this into a separate PR.

Reply:
Reverted it for this branch.